libs/sdk-core/src/logger.rs

```diff
@@ -0,0 +1,185 @@
+use crate::models::LevelFilter as BindingLevelFilter;
+use anyhow::{anyhow, Result};
+use env_logger::{Builder, Logger, Target};
+
+use log::{
+    max_level, set_boxed_logger, set_max_level, LevelFilter, Log, Metadata, Record,
+    STATIC_MAX_LEVEL,
+};
+
+use std::fs::OpenOptions;
+use std::io::Write;
+
+use chrono::Utc;
+
+/* env_logger */
+
+const ENV_LOGGER_FILTER: &str = r#"
+debug,
+breez_sdk_core::backup=info,
+breez_sdk_core::breez_services=info,
+breez_sdk_core::input_parser=warn,
+breez_sdk_core::persist::reverseswap=info,
+breez_sdk_core::reverseswap=info,
+sdk_common=debug,
+gl_client::node=info,
+gl_client::node::service=info,
+gl_client::tls=info,
+gl_client::scheduler=info,
+gl_client::signer=info,
+gl_client=debug,
```

Comment:
If I understand correctly, `debug` level logs would be filtered out regardless of the client's loglevel. I would put all these at debug, or even trace, because they would eventually be filtered out by the client log level.